.appveyor.yml

```diff
@@ -1,12 +0,0 @@
-image:
-  - Visual Studio 2019
-skip_commits:
-  message: /\[skip appveyor\]/
-clone_folder: c:\projects\source
-build_script:
-- cmd: >-
-    mkdir build &&
-    cd build &&
-    cmake c:\projects\source -DKokkos_ENABLE_TESTS=ON -DCMAKE_CXX_FLAGS="/W0 /EHsc" -DKokkos_ENABLE_DEPRECATED_CODE_4=ON &&
```

Comment:
Don't we want to keep the flags and deprecated code we had before?

Reply:
They caused issues, I don't have the inclination to investigate why now. And this is blocking the View progress.